File: Serene/Serene.Script/Meeting/MeetingLocation/MeetingLocationGrid.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System;
    using System.Collections.Generic;
    using System.Runtime.CompilerServices;

    [ColumnsKey("Meeting.MeetingLocation"), IdProperty("MeetingLocationId"), NameProperty("Name"), IsActiveProperty("IsActive")]
    [DialogType(typeof(MeetingLocationDialog)), LocalTextPrefix("Meeting.MeetingLocation"), Service("Meeting/MeetingLocation")]
    public class MeetingLocationGrid : EntityGrid<MeetingLocationRow>
    {
        public MeetingLocationGrid(jQueryObject container)
            : base(container)
        {
        }
    }

    // Please remove this partial class or the first line below, after you run ScriptContexts.tt
    [Imported, Serializable, PreserveMemberCase] 
    public partial class MeetingLocationRow
    {
    }
}

File: Serene/Serene.Core/Modules/Meeting/MeetingLocation/MeetingLocationGrid.ts

namespace Serene.Meeting {
    
    @Serenity.Decorators.registerClass()
    export class MeetingLocationGrid extends Serenity.EntityGrid<MeetingLocationRow, any> {
        protected getColumnsKey() { return 'Meeting.MeetingLocation'; }
        protected getDialogType() { return MeetingLocationDialog; }
        protected getIdProperty() { return MeetingLocationRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingLocationRow.localTextPrefix; }
        protected getService() { return MeetingLocationService.baseUrl; }

        constructor(container: JQuery) {
            super(container);
        }
    }
}